DiamondLightSource/python3-pip-skeleton-cli
# Existing command should be best effort

Closes #45 

# To test
* Test to see if setting up a repo from existing works
    * Check that the preference order for username/email is setup.cfg -> pyproject.toml -> git (git user.name, user.email) -> user input
